### ISSUE TITLE
Monospace font for Jobs Output textarea 

### ISSUE THREAD

**Issue by Richard004** [Azure/azure-webjobs-sdk#1740]
It would be nice to use **monospace** font for the output window of Jobs and Functions.

Like this:
![image](https://user-images.githubusercontent.com/8113705/41154897-01941e08-6b1c-11e8-9155-8f2379c15b86.png)

Instead of existing default font:
![image](https://user-images.githubusercontent.com/8113705/41154904-0a349f2e-6b1c-11e8-9a6c-b8f574a73e0a.png)

I guess it is in
\azure-webjobs-sdk\src\Dashboard\app\views\shared\ConsoleOutput.html

My guess is that most of the outputs are logging-like types of output.
But feel free to cancel this issue. It is just a proposal.

Have a good day.

**Comment by vaughn-pworx** [comment]
I agree, this would be really great to help produce and consume output that is more readable. The same could be said for the Application Insights web portal, but I'm not sure where I would request that.
